### PR TITLE
Update the OPTIONS_TXT/output of make_tracks_file

### PR DESCRIPTION
**Pull request by lldelisle**
Following #235 I added the negative_color.
Following #103 I updated the doc to fit with the code.
I rechecked the output of make_tracks_file (the OPTIONS_TXT) and I realized some parameters were duplicated, some lines were very long...

### CODE REVIEW

File: pygenometracks/tracks/TADsTrack.py [hunk 1]
```diff
@@ -7,6 +8,26 @@
...
+#color = RdYlBu
+#min_value=0
+#max_value=100
+# If the color is simply a color name, then this color is used and the score is not considered.
+color = darkblue
+# optional: line_width
+#line_width = 0.5
+# optional, default is black. To remove the border, simply set 'border_color' to none
+#border_color = black
+# optional. If not given is guessed from the file ending.
```
Comment (bgruening) [inline pygenometracks/tracks/TADsTrack.py:28]: s/is/it is/

File: pygenometracks/tracks/TADsTrack.py [hunk 2]
```diff
@@ -7,6 +8,26 @@
 class TADsTrack(BedTrack):
     SUPPORTED_ENDINGS = ['.domain', '.domains', '.tad', '.tads']
     TRACK_TYPE = 'domains'
+    OPTIONS_TXT = GenomeTrack.OPTIONS_TXT + """
+# If the bed file contains a column for color (column 9), then this color can be used by
+# setting:
+# color = bed_rgb
+#if color is a valid colormap name (like RbBlGn), then the score (column 5) is mapped
```
Comment (bgruening) [inline pygenometracks/tracks/TADsTrack.py:15]: `s/#if/# if/`

File: pygenometracks/tracks/BedTrack.py [hunk 3]
```diff
@@ -32,7 +32,7 @@ class BedTrack(GenomeTrack):
 # If the bed file contains a column for color (column 9), then this color can be used by
 # setting:
 # color = bed_rgb
-#if color is a valid colormap name (like RbBlGn), then the score is mapped
+#if color is a valid colormap name (like RbBlGn), then the score (column 5) is mapped
```
Comment (bgruening) [inline pygenometracks/tracks/BedTrack.py:35]: space missing?

File: docs/content/tracks/auto/bed_options_text.txt [hunk 4]
```diff
@@ -15,7 +16,7 @@ height = 2
 # If the bed file contains a column for color (column 9), then this color can be used by
 # setting:
 # color = bed_rgb
-#if color is a valid colormap name (like RbBlGn), then the score is mapped
+#if color is a valid colormap name (like RbBlGn), then the score (column 5) is mapped
```
Comment (bgruening) [inline docs/content/tracks/auto/bed_options_text.txt:19]: space missing

**Comment by lldelisle** [comment]
Thanks, all done

**Pull request merged**